Resources/Prototypes/Catalog/Fills/Crates/emergency.yml

```diff
@@ -27,7 +27,7 @@
         amount: 2
       - id: ClothingOuterSuitFire
         amount: 2
-      - id: YellowOxygenTankFilled
+      - id: OxygenTankFilled
```

Comment:
It can be removed here, this is fine. Just don't break prototype ID's needlessly.